### PR TITLE
COR-350 fix(clerk-js): Add password bytes exceeded error message

### PR DESCRIPTION
**Pull request by raptisj**
## Type of change

- [ ] 🐛 Bug fix
- [ ] 🌟 New feature
- [ ] 🔨 Breaking change
- [x] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

## Packages affected

- [x] `@clerk/clerk-js`
- [ ] `@clerk/clerk-react`
- [ ] `@clerk/nextjs`
- [ ] `@clerk/remix`
- [x] `@clerk/types`
- [ ] `@clerk/themes`
- [x] `@clerk/localizations`
- [ ] `@clerk/clerk-expo`
- [ ] `@clerk/backend`
- [ ] `@clerk/clerk-sdk-node`
- [ ] `@clerk/shared`
- [ ] `@clerk/fastify`
- [ ] `@clerk/chrome-extension`
- [ ] `gatsby-plugin-clerk`
- [ ] `build/tooling/chore`

## Description

- [ ] `npm test` runs as expected.
- [ ] `npm run build` runs as expected.

Add password bytes exceeded error message

### CODE REVIEW

File: packages/localizations/src/en-US.ts [hunk 1]
```diff
@@ -576,6 +576,8 @@ export const enUS: LocalizationResource = {
     form_identifier_exists: '',
     form_password_validation_failed: 'Incorrect Password',
     form_password_not_strong_enough: 'Your password is not strong enough.',
+    form_password_size_in_bytes_exceeded:
+      'Your password has exceeded the maximum number of bytes allowed, please shorten it or remove some special characters.',
```
Comment (panteliselef) [inline packages/localizations/src/en-US.ts:580]: ❓I understand that fapi will have to throw an error in that case, but why does the end user need to know about "bytes" ? The term feels quite technical for someone who just want to sign up. 

Should we try something like "Invalid password, please shorten it or remove some special characters."

File: packages/localizations/src/en-US.ts [hunk 2]
```diff
@@ -565,7 +565,7 @@ export const enUS: LocalizationResource = {
   unstable__errors: {
     form_identifier_not_found: '',
     form_password_pwned:
-      'This password has been found as part of a breach and can not be used, please try another password instead',
+      'This password has been found as part of a breach and can not be used, please try another password instead.',
```
Comment (panteliselef) [inline packages/localizations/src/en-US.ts:568]: 💯

**Review by gkats — APPROVED** [review]
Why is there a change in `package-lock.json`? Seems unrelated.

Other than that, looks good to me.

File: packages/localizations/src/en-US.ts (hunk 1 above)
Comment (raptisj) [reply packages/localizations/src/en-US.ts:580]: @panteliselef yes it seems odd

> Why is there a change in `package-lock.json`? Seems unrelated.
> 
> Other than that, looks good to me.

This resulted after pulling the latest main branch and doing `npm install`.  cc @nikosdouvlis do you think this is an issue?

**Comment by raptisj** [comment]
> Why is there a change in `package-lock.json`? Seems unrelated.
> 
> Other than that, looks good to me.

This resulted after pulling the latest main branch and doing `npm install`.  cc @nikosdouvlis do you think this is an issue?

**Review by panteliselef — APPROVED** [review]

**Comment by nikosdouvlis** [comment]
@raptisj I just rebased from main, this is good to go.

**Pull request merged**